codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2Connection.java

```diff
@@ -809,6 +813,7 @@ private void lastKnownStream(int lastKnownStream) {
             if (!alreadyNotified) {
                 notifyGoingAway();
             }
+            closeStreamsGreaterThanLastKnownStream();
```

Comment:
@buchgr now that we're adding a side-effect to `Http2Connection.goAwaySent` and `Http2Connection.goAwayReceived`, we should document it in the interface.